packages/context-menu/src/vaadin-context-menu-item.d.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2016 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { Item } from '@vaadin/item/src/vaadin-item.js';
+
+/**
+ * An element used internally by `<vaadin-context-menu>`. Not intended to be used separately.
+ *
+ * @protected
+ */
+declare class ContextMenuItem extends Item {}
```

Comment:
This class isn't exported because it's considered internal. Also, its name collides with `ContextMenuItem` type.
Because of it, previously when located in the mixin file, it was named `ContextMenuItemElement`. 

I think we can keep these two classes as "private" for now, because they don't enhance parent classes with any logic. Generally, such components as `vaadin-select-item` do not provide type definitions at all.

Reply:
Well, at least `vaadin-context-menu-item` is used by the Flow counterpart, so they aren't entirely private, but types don't make sense for Flow, I agree.